### PR TITLE
Fixing nil pointer error

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/pkg/operations/upload.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func UploadArtifacts(r *releasetypes.ReleaseConfig, eksArtifacts map[string][]re
 			}
 
 			// If the artifact is a helm chart, skip the skopeo copy as it's handled separately.
-			if strings.HasSuffix(artifact.Image.AssetName, "helm") {
+			if artifact.Image.AssetName != "" && strings.HasSuffix(artifact.Image.AssetName, "helm") {
```
Comment (abhay-krishna) [inline release/pkg/operations/upload.go:84]: Can you move this into the if block on line 88?

Comment (abhay-krishna) [reply release/pkg/operations/upload.go:84]: And you can remove the empty string check.

Comment (TerryHowe) [reply release/pkg/operations/upload.go:84]: Is AssetName nil, because I think len(Assetname) == 0 is true for both "" and nil

Comment (jonahjon) [reply release/pkg/operations/upload.go]: updated!

Comment (jonahjon) [reply release/pkg/operations/upload.go]: I can remove the empty string check, but lemme doublecheck that works still.

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**